src/main/java/data/Database.java

```diff
@@ -1,38 +1,43 @@
 package data;
 
+import java.io.File;
 import java.io.FileInputStream;
 import java.io.FileNotFoundException;
 import java.io.FileOutputStream;
 import java.io.InputStream;
 import java.io.OutputStream;
-import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
 
 /* This class creates an instance of a database that a file can be written to and read from.
 Assumptions: for each runID, only 1 of each datatype can be associated with it. */
 public class Database {
-  /* Place where files are stored.*/
-  private String directory;
 
-  /* List of all files stored in Database.*/
-  private ArrayList<String> files;
+  /* Folder for Database. */
+  File folder;
 
   /* Creates instance of a database with this name. */
   public Database(String name) {
-    this.directory = name;
-    files = new ArrayList<String>();
+    folder = new File(name);
+    folder.mkdir();
   }
 
   /* Returns name associated with Database. */
-  public String getDatabase() {
-    return directory;
+  public File getDatabase() {
+    return folder;
```

Comment:
i made public here bc i was think about how it could be accessed once the database was abstracted when we switch from b/w a file system and a GCS, but i realize now there is a way to make a database interface and still had a file database class that has this method hidden in it

Reply:
This isn't resolved; I'm still not sure why the `File` is exposed to users of `Database`. It looks like the function is only used in tests to clean up. Maybe you can write a `delete` method on `Database` instead?